File: bindings/swift/smoke/SmokeTest.swift
import Gurmukhi

@main
struct SmokeTest {
    static func main() {
        let result = toUnicode(input: "gurU", standard: .santLipi)
        guard result == "ਗੁਰੂ" else {
            fatalError("FAIL: expected ਗੁਰੂ, got \(result)")
        }
        print("PASS: Swift")
    }
}

File: bindings/kotlin/smoke/SmokeTest.kt
import uniffi.gurmukhi.toUnicode
import uniffi.gurmukhi.UnicodeStandard

fun main() {
    val result = toUnicode("gurU", UnicodeStandard.SANT_LIPI)
    check(result == "ਗੁਰੂ") { """FAIL: expected "ਗੁਰੂ", got "$result"""" }
    println("PASS: Kotlin")
}
